must-gather/collection-scripts/gather

```diff
@@ -19,9 +19,8 @@ gather_clusterscoped_resources ${BASE_COLLECTION_PATH} "${SINCE_TIME}"
 gather_noobaa_resources ${BASE_COLLECTION_PATH} "${SINCE_TIME}"
 
 storageClusterPresent=$(oc get storagecluster -n openshift-storage -o go-template='{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}')
-externalStorageClusterPresent=$(oc get storagecluster -n openshift-storage -o go-template='{{range .items}}{{.spec.externalStorage.enable}}{{"\n"}}{{end}}')
 
-if [[ -n "${externalStorageClusterPresent}" ]]; then
+if [ "$(oc get storagecluster -n openshift-storage -o go-template='{{range .items}}{{.spec.externalStorage.enable}}{{"\n"}}{{end}}')" == true ]; then
```

Comment:
I reverted this change to the previous one since the output for `if [[ -n "${externalStorageClusterPresent}" ]]` was `<no value>` which is a non-empty string, so the must gather script kept skipping due to this.

Reply:
why it is no value?